src/foundry/client/config.d.mts

```diff
@@ -542,6 +607,9 @@ declare global {
       /** @defaultValue `0.25` */
       darknessLightPenalty: number;
 
+      /** @defaultValue `/Edg|Firefox|Electron/` */
```

Comment:
Is it really only "Edg" in the regex, not "Edge"?

Reply:
Yes that was a direct copy paste